### PR TITLE
fix: broken contents

### PR DESCRIPTION
**Pull request by caramis**

### CODE REVIEW

**Comment by caramis** [comment]
The work is still in progress.

File: content/docs/develop/get-started.md [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
 ### Other Tools
 
 Xpla also provides other tools for building:
 
 - [Finder](https://finder.c2x.world): Xpla’s multi-purpose block explorer. 
 - [Faucet](https://faucet.c2x.world): Get tokens sent to your testnet address. 
-- [`xplad`](xplad/install-xplad.md): the node daemon and command line interface that connects to the Xpla network. 
+- [`xplad`]({{< ref "install-xplad" >}}): the node daemon and command line interface that connects to the Xpla network. 
 
 ## Resources
```
Comment (jhlee-young) [inline content/docs/develop/get-started.md:37]: no needs for few resources below?

File: content/docs/develop/guides/tictaetoe-app-example.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
```
Comment (jhlee-young) [inline content/docs/develop/guides/tictaetoe-app-example.md]: we cannot use this example. seems better to delete the page at all

File: content/docs/develop/core-modules/evidence.md [hunk 3]
```diff
@@ -55,7 +56,7 @@ type ValidatorEvidence interface {
 }
 ```
```
Comment (jhlee-young) [inline content/docs/develop/core-modules/evidence.md:57]: need to specify the language here and below also

File: content/docs/develop/core-modules/fee-grant.md [hunk 4]
```diff
@@ -21,7 +22,7 @@ Every `grant` contains the following information:
 
 - `granter`: The account address that gives permission to the grantee.
 - `grantee`: The beneficiary account address.
-- `allowance`: The [type of fee allowance](#fee-allowance-types) given to the grantee. `Allowance` accepts an interface that implements `FeeAllowanceI` encoded as `Any` type, as shown in the following example:
+- `allowance`: The [type of fee allowance]({{< ref "#fee-allowance-types" >}}) given to the grantee. `Allowance` accepts an interface that implements `FeeAllowanceI` encoded as `Any` type, as shown in the following example:
 
   ```
     // allowance can be any of basic and filtered fee allowance.
```
Comment (jhlee-young) [inline content/docs/develop/core-modules/fee-grant.md:28]: also needs lang on this block

File: content/docs/develop/tools/xpla-js/_index.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
 bookCollapseSection: true
 bookFlatSection: false
-weight: 110
-title: Xpla js
+weight: 30
+title: Xpla.js
 ---
```
Comment (jhlee-young) [inline content/docs/develop/tools/xpla-js/_index.md:7]: remaining not supported tag below

File: content/docs/develop/tools/xpla-js/getting-started.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-weight: 50
+weight: 10
 title: Getting Started
 ---
```
Comment (jhlee-young) [inline content/docs/develop/tools/xpla-js/getting-started.md:5]: remaining unsupported tags below. plz update headers to start with capital letter.

File: content/docs/develop/tools/xpla-js/common-examples.md [hunk 7]
```diff
@@ -1,15 +1,16 @@
 ---
-weight: 30
+weight: 20
 title: Common Examples
 ---
 
-# Common Examples Guide
+# Common Examples
 
 Use the following common examples to learn how to use xpla.js. If this is your first time using xpla.js, use the [xpla.js installation guide](./getting-started.md).
```
Comment (jhlee-young) [inline content/docs/develop/tools/xpla-js/common-examples.md]: a broken link to the guide

**Review by yoosah — COMMENTED** [review]
xplad/commands.md:196 should be changed to `xplad tendermint unsafe-reset-all --home </path-to-nodehome>

**Comment by psy2848048** [comment]
Can't point into code diff but all links are broken in `/docs/learn/fees/` page

**Comment by psy2848048** [comment]
`/docs/learn/glossary/#active-set` no change the number: 130 validators?

**Comment by psy2848048** [comment]
`/docs/learn/glossary/#fees` broken link in "visit `fees`"

**Comment by psy2848048** [comment]
`/docs/learn/glossary/#slashing` broken link in `slashing`

**Comment by psy2848048** [comment]
`/docs/learn/glossary/#staking` broken link of `concepts page`

**Comment by psy2848048** [comment]
`/docs/learn/glossary/#staking` broken link in `Xpla core modules`

**Comment by psy2848048** [comment]
`/docs/learn/glossary/#station` broken links

**Comment by psy2848048** [comment]
`/docs/learn/glossary/#station` broken link

**Comment by psy2848048** [comment]
`/docs/learn/glossary/#station` broken link

**Comment by psy2848048** [comment]
`/docs/learn/network/#what-is-xpla` broken link in `staking` and `blockchain`

**Comment by psy2848048** [comment]
`/docs/learn/network/#validators` all 2 links are broken

**Comment by psy2848048** [comment]
/docs/learn/network/#staking broken link `station staking guide`

**Comment by psy2848048** [comment]
/docs/learn/network/#delegators 
1. broken md `:::{admonition} Who owns staked XPLA? :class: warning`
1. broken link `Validator FAQ`

**Comment by psy2848048** [comment]
`/docs/learn/network/#bonding-staking-and-delegating` broken link `active set`

**Comment by psy2848048** [comment]
`/docs/learn/network/#unbonding` broken MD

**Comment by psy2848048** [comment]
`/docs/learn/network/#redelegation` broken MD `:::{caution}`

**Comment by psy2848048** [comment]
`/docs/learn/network/#rewards` all broken links

**Comment by psy2848048** [comment]
`/docs/learn/network/#slashing` 
1. broken link 
2. broken MD

**Comment by psy2848048** [comment]
`/docs/learn/network/#governance` broken link

**Comment by psy2848048** [comment]
`/docs/learn/station/` broken link

**Comment by psy2848048** [comment]
`/docs/learn/station/contracts/` broken image reference beside the headline `Contract`

**Comment by psy2848048** [comment]
`/docs/learn/station/download/station-extension/#create-a-wallet` link should be rewrited into Xpla

**Comment by psy2848048** [comment]
`/docs/learn/station/governance/`
1. broken image on the headline
2. broken links

File: content/docs/develop/get-started.md (hunk 1 above)
Comment (caramis) [reply content/docs/develop/get-started.md:37]: What resouces ??

File: content/docs/develop/guides/tictaetoe-app-example.md (hunk 2 above)
Comment (caramis) [reply content/docs/develop/guides/tictaetoe-app-example.md]: removed !

File: content/docs/develop/core-modules/evidence.md (hunk 3 above)
Comment (caramis) [reply content/docs/develop/core-modules/evidence.md:57]: What a great review !!
Applied !

**Comment by psy2848048** [comment]
`/docs/learn/station/governance/#deposit-xpla-to-a-proposal`
`/docs/learn/station/governance/#create-a-new-proposal`
broken MD `:::{caution}`

File: content/docs/develop/core-modules/fee-grant.md (hunk 4 above)
Comment (caramis) [reply content/docs/develop/core-modules/fee-grant.md:28]: Applied !

File: content/docs/develop/tools/xpla-js/_index.md (hunk 5 above)
Comment (caramis) [reply content/docs/develop/tools/xpla-js/_index.md:7]: 👍

**Comment by psy2848048** [comment]
`/docs/develop/get-started/#get-started`
Learn about Xpla: link is good but no content inside of there

File: content/docs/develop/tools/xpla-js/getting-started.md (hunk 6 above)
Comment (caramis) [reply content/docs/develop/tools/xpla-js/getting-started.md:5]: Fixed

File: content/docs/develop/tools/xpla-js/common-examples.md (hunk 7 above)
Comment (caramis) [reply content/docs/develop/tools/xpla-js/common-examples.md]: fixed

**Review by jhlee-young — APPROVED** [review]
LGTM

**Comment by psy2848048** [comment]
`/docs/develop/guides/smart-contract-guide/interacting-with-the-contract/#uploading-code`

This section should be placed the right after `/docs/develop/guides/smart-contract-guide/writing-the-contract/`

**Comment by psy2848048** [comment]
`/docs/develop/guides/tictaetoe-app-example/`

Good to delete the deployed website & code ID reference. The content seems fully integrated with other network

**Comment by psy2848048** [comment]
`/docs/develop/tools/xpla-js/getting-started/#2-initialize-the-lcd` broken md `::: {admonition}`

**Comment by psy2848048** [comment]
`/docs/develop/tools/xpla-js/getting-started/#3-create-a-tesseract-testnet-wallet` broken md `:::{admonition}`

**Comment by psy2848048** [comment]
`/docs/develop/tools/xplad/install-xplad/#install-xplad` end of the paragraph, missing any link after `visit .`

**Comment by psy2848048** [comment]
/docs/develop/tools/xplad/using-xplad/#fees
broken formula `$$fees = gas * gasPrices$$`

**Comment by psy2848048** [comment]
`/docs/develop/core-modules/capability/#initialization`
All snippets are not golang colored

**Comment by psy2848048** [comment]
`/docs/develop/core-modules/evidence/#concepts`
`/docs/develop/core-modules/evidence/#registration-and-handling`
`/docs/develop/core-modules/evidence/#messages`
`/docs/develop/core-modules/fee-grant/`

All snippets are not golang colored

**Comment by psy2848048** [comment]
`/docs/develop/core-modules/slashing/#liveness-tracking` broken MD `:::{Important}`

**Comment by psy2848048** [comment]
`/docs/full-node/manage-a-validator/#manage-a-validator-img-srcimgbuild_a_validatorsvg-height40px` 
1. missing image beside of the headline
2. broken links
3. 130 active validator set -> is this fixed?
4. 7% inflation -> is this fixed?

File: content/docs/develop/tools/xplad/install-xplad.md [hunk 8]
```diff
@@ -1,30 +1,30 @@
...
+weight: 10
+title: Install xplad
 ---
 
 # Install `xplad`
 
 `xplad` is the command-line interface and daemon that connects to Xpla and enables you to interact with the Xpla blockchain. Xpla core is the official Golang reference implementation of the Xpla node software.
 
-This guide is for developers who want to install `xplad` and interact with Xpla core without running a full node. If you want to run a full node or join a network, visit [](../../full-node/run-a-full-node/_index.md).
+This guide is for developers who want to install `xplad` and interact with Xpla core without running a full node. If you want to run a full node or join a network, visit R[Run a Full Xpla Node]({{< ref "run-a-full-node" >}}).
```
Comment (jhlee-young) [inline content/docs/develop/tools/xplad/install-xplad.md]: typo at the final link :) `R[Run`

File: content/docs/develop/core-modules/crisis.md [hunk 9]
```diff
@@ -5,17 +5,18 @@ title: Crisis
...
 
-## ConstantFee
+## Constant Fee
 
 Because the gas cost of verifying an invariant is high, the crisis module utilizes a constant fee instead of the typical gas calculation method. 
 
-## Genesis parameters
+## Genesis Parameters
 
 The genesis parameters for the crisis module outlined in the [Genesis Builder Script](https://github.com/c2xdev/genesis-tools/blob/main/src/genesis_builder.py#L141) are as follows:
```
Comment (jhlee-young) [inline content/docs/develop/core-modules/crisis.md:22]: could you update the amount below python block? seems should be `512000000000000000000`

File: content/docs/develop/core-modules/governance.md [hunk 10]
```diff
@@ -258,7 +261,7 @@ type VotingParams struct {
 }
 ```
 
-## Genesis parameters
+## Genesis Parameters
 
 The genesis parameters for the governance module outlined in the [Genesis Builder Script](https://github.com/c2xdev/genesis-tools/blob/main/src/genesis_builder.py#L147) are as follows:
```
Comment (jhlee-young) [inline content/docs/develop/core-modules/governance.md:267]: below py block also needs amount update to 512000000000000000000

File: content/docs/develop/core-modules/_index.md [hunk 11]
```diff
@@ -1,18 +1,18 @@
...
 
 The Xpla Core is the official Golang reference implementation of the Xpla protocol.
 The Xpla Core is built using the [Cosmos SDK](https://cosmos.network/sdk), which provides a robust framework for blockchains that run atop the [Tendermint](https://tendermint.com/) consensus protocol.
 
 Before diving into the core modules, it may be useful to familiarize yourself with the [Cosmos](https://docs.cosmos.network/) and [Tendermint](https://docs.tendermint.com/master/tutorials/go.html) documentation. 
 
-## How to use the Xpla Core module specifications
+## How to Use the Xpla Core Module Specifications
 
 Each module specification begins with a short description of the module's main function within the architecture of the system and an eXplanation of how it contributes to implementing Xpla's features.
```
Comment (jhlee-young) [inline content/docs/develop/core-modules/_index.md]: eXplanation -> explanation, my fault 😅 (+ also 23 line)

File: content/docs/develop/tools/xplad/install-xplad.md (hunk 8 above)
Comment (psy2848048) [reply content/docs/develop/tools/xplad/install-xplad.md]: Related with https://github.com/c2xdev/docs/pull/2#issuecomment-1204202139
Put comment for getting the fix alarm

**Pull request merged**